includes/functions.inc.php

```diff
@@ -1670,21 +1670,30 @@ function tag_cloud($days, $scale_min, $scale_max) {
 /**
  * converts a unix timestamp into a formated date string
  *
- * @param string $format : like parameter for DateTime::format() - https://www.php.net/manual/de/datetime.format.php
- * @param int $timestamp : UNIX timestamp
+ * @param string $format : like parameter for IntlDateFormatter::format
+ * @param long $timestamp : UNIX timestamp
```

Comment:
Is it correct to mention the Unix-timestamp as long? AFAIK it is a signed 32-bit integer (see 2038-problem) and therefore "only" of type int.

Reply:
I changed the comment

Reply:
Thank you.

A general consideration: Beside the comment itself, is there a data type `long` in PHP? The only references I found in the PHP documentation belongs to IPv4-handling (`ip2long`, `long2ip`) and the function `is_long`, which is an alias of `is_int`. I know thedistinction of `int` and `long` only from ancient programming languages like VB (in its version in MS Office 97), where `int` had a value range of -32768 to 32767 and `long` was the equivalent of `int` in modern programming languages.

Reply:
Normally I program in Java and here a timestamp is a 64 bit long type. My fault.